src/gui/application.cpp

```diff
@@ -186,7 +186,8 @@ Application::Application(int &argc, char **argv)
     setAttribute(Qt::AA_UseHighDpiPixmaps, true);
 
     auto confDir = ConfigFile().configPath();
-    if (!QFileInfo(confDir).exists()) {
+    if (confDir.endsWith('/')) confDir.chop(1);  // macOS 10.11.x does not like trailing slash for rename/move.
```

Comment:
Note:  confDir ALWAYS ends in a slash  (by construction)

Reply:
Yep, if this ever changes then this migration doesnt break :) So I prefer to keep the checks explicit.